### ISSUE TITLE
Should the install_start watchdog be 3 hours?

### ISSUE THREAD

**Issue by jvillal-amp** [beaker-project/beaker#61]
So the install_start watchdog appears to be set to 3 hours when the install starts. Which seems kind of long for the use cases I do.

Are installs really taking longer than 30 minutes?

The watchdog gets set to 3 hours when the install process tells the beaker lab controller that it is starting the install. So the install has started at that point.

The reason I am asking is we are sometimes seeing issues where our install finishes reboots and then something goes wrong in our firmware and it doesn't boot from the disk. So then it takes about 3 hours for the job to get aborted.

Looking at the watchdog flow it appears to be:
  * 30 minutes when the provision process starts
  * 3 hours when the install kickstart hits the "/install_start/<recipe_id>" on the lab controller
  * 40 minutes (2400 seconds) when Restraint starts
  * Then task specific watchdog time once task starts.

install_start code:
 * https://github.com/beaker-project/beaker/blob/87a0f32559769f4a7898445718b2251b511466ae/LabController/src/bkr/labcontroller/proxy.py#L634-L640
 * https://github.com/beaker-project/beaker/blob/87a0f32559769f4a7898445718b2251b511466ae/Server/bkr/server/recipes.py#L217-L235

**Comment by StykMartin** [comment]
hi @jvillal-amp,

tl;dr - Yes. The values are Correct.

Long one.  All values were precisely picked based on systems used in Red Hat and issues with NFS / HTTP / FTP services servicing all the content. It may look like overkill but we bridged some of the values in the past.

If your system is connected to console, then make sure that the Install detector + Panic detector are turned on. It will abort installation in case you will meet criterium.
